pro/build-nextflow-schemas/action.yml

```
@@ -106,8 +110,12 @@ runs:
           outputs+=(--output "$component_outputs_schema")
         fi
 
-
-        echo "Running viash_tools/target/docker/nextflow/generate_schema/generate_schema ${inputs[@]} ${outputs[@]}"
-        viash_tools/target/docker/nextflow/generate_schema/generate_schema "${inputs[@]}" "${outputs[@]}"
-        echo "Created $outputs_schema"
-      
\ No newline at end of file
+        if [ "${{ inputs.enable_dataset_input }}" ]; then
```

I don't think this will work... you might need the following suggestion. I remember that I used to have problems with this in github actions in the past.

```suggestion
        if [ "${{ inputs.enable_dataset_input }}" == "true" ]; then
```

---

ran a test build in OP and it worked without the string conversion as well, but can convert it to avoid future trouble